slick/src/main/scala/slick/ast/Comprehension.scala

```diff
@@ -9,9 +9,11 @@ final case class Comprehension(sym: TermSymbol, from: Node, select: Node, where:
                                groupBy: Option[Node] = None, orderBy: ConstArray[(Node, Ordering)] = ConstArray.empty,
                                having: Option[Node] = None,
                                distinct: Option[Node] = None,
-                               fetch: Option[Node] = None, offset: Option[Node] = None) extends DefNode {
+                               fetch: Option[Node] = None,
+                               offset: Option[Node] = None,
+                               forUpdate: Option[Node] = None) extends DefNode {
```

Comment:
This is to support the syntax which specifies explicit columns to lock to give finer grained locking control e.g. ```for update of col1, col2``` http://docs.oracle.com/cd/E11882_01/server.112/e41084/statements_10002.htm#SQLRF55372. As I mentioned in the text for the PR, I haven't implement this yet, but by implementing it as a node, gives us placeholder to add it.